### PR TITLE
VUE-388 Auto Deposit FAQ

### PR DESCRIPTION
**Pull request by eddieferrer**
* Add Auto Deposit FAQ section and organize some components

### CODE REVIEW

File: src/pages/PageTwo.vue [hunk 1]
```diff
@@ -91,7 +91,7 @@ import EmailPreviewContentful from '@/pages/MonthlyGood/EmailPreviewContentful';
 
 import MoreAboutKiva from '@/pages/MonthlyGood/MoreAboutKiva';
 import KivaAsExpertContentful from '@/pages/MonthlyGood/KivaAsExpertContentful';
-import FrequentlyAskedQuestions from '@/pages/MonthlyGood/FrequentlyAskedQuestions';
+import FrequentlyAskedQuestions from '@/components/MonthlyGood/FrequentlyAskedQuestions';
```
Comment (eddieferrer) [inline src/pages/PageTwo.vue:94]: I prefer this organization where components in /pages/ are referenced only in the router. Hence why I moved this FAQ compoennt to /components/

File: src/components/AutoDeposit/FrequentlyAskedQuestions.vue [hunk 2]
```diff
@@ -0,0 +1,78 @@
+<template>
```
Comment (eddieferrer) [inline src/components/AutoDeposit/FrequentlyAskedQuestions.vue:1]: This is a separate component, almost identical to the MG FAQ component. Future improvement would be to make these 2 components get content from contentful based on a prop key.

File: src/components/MonthlyGood/FrequentlyAskedQuestions.vue [hunk 3]
```diff
@@ -21,16 +21,15 @@
...
 
 export default {
 	components: {
-		ExpandableQuestion
+		KvExpandableQuestion
 	},
 	data() {
 		/* eslint-disable max-len */
 		return {
-			open: false,
```
Comment (eddieferrer) [inline src/components/MonthlyGood/FrequentlyAskedQuestions.vue:33]: This is unused, probably left here by me 😞

File: src/components/AutoDeposit/FrequentlyAskedQuestions.vue (hunk 2 above)
Comment (mcstover) [reply src/components/AutoDeposit/FrequentlyAskedQuestions.vue:1]: Yea we should make an FAQ component someday!

**Pull request merged**